wordpress-mobile/WordPress-iOS
# Fixes the internal icon generation script.

## Description

This PR fixes a problem with the icon generation script for internal builds.

Changes:
- The Icon Generation Script now runs before resources are copied.
- The internal iconset is once more generated in the project directory.
- The internal iconset is now gitignored.

## Testing

1. Update the file `WordPress-Internal-Info.plist` and set the short version to `8.9` and the long version to `8.9.20171122`.
2. Archive an internal build.
3. Upload to hockeyapp and restrict downloads to yourself (and eventually other testers).
4. Download and check the App Icon is fine.
